app/src/main/java/co/nimblehq/compose/crypto/ui/screens/home/HomeFragment.kt

```diff
@@ -0,0 +1,24 @@
+package co.nimblehq.compose.crypto.ui.screens.home
+
+import androidx.compose.material.ExperimentalMaterialApi
+import androidx.compose.runtime.Composable
+import co.nimblehq.compose.crypto.ui.base.BaseComposeFragment
+import co.nimblehq.compose.crypto.ui.screens.compose.composables.home.HomeScreen
+import co.nimblehq.compose.crypto.ui.screens.compose.theme.ComposeTheme
+import dagger.hilt.android.AndroidEntryPoint
+
+@AndroidEntryPoint
+@ExperimentalMaterialApi
+class HomeFragment: BaseComposeFragment() {
```

Comment:
Good point 👍  @hoangnguyen92d, I haven't thought about this. So that we will use only one activity (**MainActivity**) to control every compose's screen. Totally, I agree with you. I think we may need to remove the navigation component if we don't use it. What do you think?

Reply:
@Wadeewee now we can set up navigation component with compose to manage screen stacks 😉 [This example](https://github.com/android/compose-samples/blob/master/JetNews/app/src/main/java/com/example/jetnews/ui/JetnewsNavGraph.kt) may help you somehow 🧋

Reply:
Yeah, @manh-t, we can set up a navigation component with compose. I have seen some examples in the code lab and from the `E-commerce` Repo from @hoangnguyen92dn, too 😃  but It seems like the navigation is not an issue here. We need to use the `navigation component with compose` if we don't use fragments.

>  I think we may need to remove the navigation component if we don't use it

Actually, I mean the `navigation` in `res` including `BaseNavigator`, `NavigationEvent` and `NavigationException`. I think for the XML file, we can remove it, but for those of the Kotlin files, we might be updated somehow or remove it. 🙏

Reply:
@hoangnguyen92dn I have one more concern about VM. If we put every screen in `MainActivity` So, we might have many VMs in Activity 🤔 Could you please share the idea with me 🙏

Reply:
Oh, I just noticed the part of VM on [JetNews](https://github.com/android/compose-samples/blob/251fbc6262ecf255a1bc3d20e0fd1d4760ef232a/JetNews/app/src/main/java/com/example/jetnews/ui/JetnewsNavGraph.kt#L46-L55) @hoangnguyen92dn. I got it now 👍 

> Or we could use DI with [Hilt](https://developer.android.com/jetpack/compose/libraries#hilt).

I think we should go with this way 🤔  So would it be better if we separate PR to handle this and remove the old implementation, What do you think?

Reply:
@Wadeewee Yeah. Please add a TODO for the part need to be updated 🙏